### ISSUE TITLE
E131: inconsistent reporting when comment present on initial line

### ISSUE THREAD

**Issue by fofoni** [PyCQA/pycodestyle#1055]
This (correctly, IMO) produces no warnings:
```python
import foo

dct = {
    'key':
        foo.yadayada() if foo.some_condition()  # raises no warnings
        else foo.blablebli()
}
```

On the other hand, simply adding a comment on the 'key' line makes it raise E131 on the two most indented lines:
```python
import foo

dct = {
    'key':  # some comment added
        foo.yadayada() if foo.some_condition()  # now, this raises E131
        else foo.blablebli()
}
```

Using pycodestyle 2.8.0.
